### ISSUE TITLE
Usability: NVDA is not reciting the when "Go to definition" and "Peek definition" is applied to the code. 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/vscode#17244]
VS Code Version: 1.6.1
OS Version:1607
NVDA version:-2016.2.1

**Steps to Reproduce:**
Prerequisite: Node.js and git repository need to be installed.
1.Open Visual studio code editor.
2.open any file and using "Ctrl+1" key  navigate to the  first vs code editor.
3.Open one file navigate to some method and  use F12 to activate "Go to Definition" in the code for same method in the code.
4.Open one file navigate to some method in the code and  use ALT + F12 to activate "Peek Definition" to the code.

**Expected result:**
1.NVDA should announce when "go to definition " is open in a separate tab. 
2.NVDA should announce  the pop up window when "Peek Definition" is opened.

**Actual Result:**
NVDA is not reciting any proper information about the code when "Go to definition" and "Peek  Definition" are opened.

**User Impact:**
Screen Reader user doesn't get to know about the General Definition and Peek Definition of the Code 

**Attachments:**
![nvda_speechviewerinformation_vscodeeditor_codedefinition](https://cloud.githubusercontent.com/assets/22956130/21213476/987c7a2e-c2ba-11e6-943a-0e958f5acc98.jpg)
